controllers/flowlogspipeline/metrics_definitions/namespace_flows_total.yaml

```diff
@@ -21,7 +21,7 @@ visualization:
   type: grafana
   grafana:
     - expr: 'topk(5,rate(netobserv_namespace_flows_total[1m]))'
-      legendFormat: '{{SrcK8S_Namespace}} : {{DstK8S_Namespace}}'
+      legendFormat: '{{pod}} : {{DstK8S_Namespace}}'
```

Comment:
For all legendFormat, what about using the same format as console plugin ?
When namespace is available`{{namespace}}.{{name}}` (resource / owner) else only `{{name}}` (namespace / host / ip) for every peers
https://github.com/netobserv/network-observability-console-plugin/blob/68239b88737f676152a9cf51e3b58bc6b781bd75/web/src/utils/metrics.ts#L77

Then we use `->` between source and destination as:
`{{source}} -> {{destination}}`
https://github.com/netobserv/network-observability-console-plugin/blob/fe28d186fcd9acc1078c54db7690e86d6c83a293/web/src/components/metrics/metrics-helper.tsx#L147

Here is the result:
![image](https://user-images.githubusercontent.com/91894519/218978945-d94b6fd7-2806-496b-a194-41316d775ceb.png)

Reply:
I added `->` between source and destination. Instead of `.` separating `namespace` and `owner` I used `,` because sometimes the name of an owner or namespace already includes `.` .

I included the `pod` in the legend because we get separate statistics for each flowlogs-pipeline instance that is running. Since the different FLPs report separately and have different labels, we do not have a cumulative summed total, since the same metrics may be spread across multiple FLPs. We have to think about how we want to handle this.

We can further change the legends in a future PR by simply changing the metrics_definitions yaml files, with no additional changes in the code.

Reply:
Yes it's good enough to merge as is. +1 for followup on totals

 Thanks @KalmanMeth !